#! /bin/sh /usr/share/dpatch/dpatch-run
## 92_125-customer_id.dpatch by  <seneca@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changing customer_id to ${vc}_id in query

@DPATCH@

diff -Naur ledgersmb-1.2.5-new/LedgerSMB/Form.pm ledgersmb-1.2.5-dirty/LedgerSMB/Form.pm
--- ledgersmb-1.2.5-new/LedgerSMB/Form.pm	2007-05-17 11:26:32.000000000 -0400
+++ ledgersmb-1.2.5-dirty/LedgerSMB/Form.pm	2007-05-28 18:02:25.000000000 -0400
@@ -2416,7 +2415,7 @@
 			ct.notes, 
 			ct.curr AS currency
 		FROM $vc ct
-		WHERE ct.id = (select customer_id from $arap where $where AND ${vc}_id IS NOT NULL order by id DESC limit 1)|;
+		WHERE ct.id = (select ${vc}_id from $arap where $where AND ${vc}_id IS NOT NULL order by id DESC limit 1)|;
 
     $sth = $dbh->prepare($query);
     $sth->execute() || $self->dberror($query);
